app/src/app/app.component.ts

```diff
@@ -16,26 +16,29 @@ export class AppComponent implements OnInit {
 
   public $latestSeedId: Observable<string>;
 
+  private readonly lastChangelogEntry = "viewed-0.28.0-changelog";
+  private readonly cutoffDate = new Date(2024, 11, 1)
```

Comment:
Should the date be updated for the release?

Reply:
Pushed an update to init the date with a string so it's less error-prone since JS dates are awful